maintainers/maintainer-list.nix

```diff
@@ -2923,10 +2923,10 @@
     name = "Robin Gloster";
   };
   gnidorah = {
-    email = "gnidorah@yandex.com";
+    email = "gnidorah@users.noreply.github.com";
```

Comment:
would you mind making this change in a separated commit?

Reply:
@marsam Done, thanks!